src/remote/activitypub/models/note.ts

```diff
@@ -301,6 +301,10 @@ export async function extractEmojis(tags: IObject | IObject[], host: string): Pr
 
 	if (!tags) return [];
 
+	// ブロックしてたら中断
+	const meta = await fetchMeta();
+	if (meta.blockedHosts.includes(host)) throw new Error('blocked instance emoji');
+
```

Comment:
You can't attach an emoji from another instance, so it may not be necessary if it's blocked by resolver.